docs/evaluation/index.mdx

```diff
@@ -2,26 +2,101 @@
 sidebar_label: Overview
 sidebar_position: 0
 ---
+import ThemedImage from '@theme/ThemedImage';
 
 # Evaluation Overview
 
-LLM applications involve putting a probablistic model at the center of your system.
-Due to the non-deterministic nature of LLMs, it is important to be able to properly evaluate your application.
-Having a proper evaluation framework that you trust will allow you to:
+## What are evaluations?
 
-- Experiment with different prompts and accurately determine if changes improve performance
-- Iterate on aspects your RAG architecture without guesswork
-- Quickly decide if it is worth upgrading to a new, recently release model
-- Determine if you can use a cheaper, faster model in place of a more expensive one and get similar performance
-- Flag with a model behind an API changed suddenly in an unexpected way
-- Have confidence your application will not produce biased or unwanted responses
+Evaluations allow you to understand the performance of your LLM application over time. At its core, an evaluation is a function that takes in a set of inputs and outputs from your chain, agent, or model, 
+and returns a score. This score may be based on comparing the outputs with reference outputs (e.g. with string matching or using an LLM as a judge).
+However, there are also evaluators that don't require a reference output - for example, one that checks if the output is blank, which is a common failure mode in LLM applications.
```

Comment:
"for example, one that checks if the output is blank, which is a common failure mode in LLM applications." maybe not all that common: maybe a more relevant one is like "output is valid json" or "conforms to specific schema"